README.md

```diff
@@ -160,6 +160,9 @@ Packages
 #### Energia [dev-embedded/energia-bin]
 [Energia](http://energia.nu/) is an electronics prototyping platform for Texas Instruments MSP430 LaunchPad based on [Wiring](http://www.wiring.org.co/) and [Arduino](https://www.arduino.cc/).
 
+#### komodo-edit-bin [app-editors/komodo-edit-bin]
```

Comment:
I'd prefer using the proper name and then the package name, i.e. `Komodo Edit [app-editors/komodo-edit-bin]`.